packages/swift/Sources/Generated.swift

```diff
@@ -15,25 +15,53 @@ public protocol Event: Codable {}
 
 public enum WidgetEvent {
     public struct Load: Event {
+
+        public static func == (lhs: WidgetEvent.Load, rhs: WidgetEvent.Load) -> Bool {
+            return true
+        }
     }
     public struct Ping: Event {
         public var userGuid: String
         public var sessionGuid: String
+
+        public static func == (lhs: WidgetEvent.Ping, rhs: WidgetEvent.Ping) -> Bool {
+            return true
```

Comment:
You'll notice that all implementations of this method start with `return true` and then go on with comparing all other fields, that's due to how I'm generating this code: since I'm iterating over all fields and appending the `&&` to that line of code, I needed something to be at the very beginning of the return expression, otherwise we'd end up with `return && ...`, which is a syntax error.

I think I may revisit this but that'll be in a future PR.

Reply:
I just updated this and this is no longer the case. The template is a little more complex now, but the generated code is less likely to be confusing.